_notices/rsn0031.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 31 # should match notice number
+notice_pin: true # set to true to pin to notice page
+title: "CUDA 12 PIP Wheels Release in v23.06"
+notice_author: RAPIDS TPM
+notice_status: In Progress
+notice_status_color: yellow
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Platform Support Change
+notice_rapids_version: "v23.06"
+notice_created: 2023-05-16
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2023-05-16
+---
+
+## Overview
+
+The core RAPIDS packages that build pip wheels (rmm, cudf, raft, cuml, cugraph, cuopt, ucx-py) are now building pip wheels with `CUDA 12 support`.  We plan to release these packages in release v23.06.

```

Comment:
I'd avoid the "core" designation here. The definition of "core" is constantly shifting, and currently we don't have pip wheels for cuSpatial which is (sometimes?) considered a core package.
```suggestion
RAPIDS packages with pip wheel support (rmm, cudf, raft, cuml, cugraph, cuopt, ucx-py) are now building pip wheels with CUDA 12 support. We plan to release these packages in release v23.06.

CUDA 12 wheels are named with a `-cu12` suffix to distinguish them from CUDA 11 wheels which have a `-cu11` suffix.
```